src/components/FeaturePanel/FeaturePanelInner.tsx

```diff
@@ -100,6 +101,8 @@ export const FeaturePanelInner = () => {
           <FeatureHeading />
           <ParentLink />
 
+          <ClimbingRestriction />
```

Comment:
@zbycz what about this? I know it shouldn't be here because it's generic FeaturePanel, but where to put it if it's for all features with climbing tag?

Reply:
It is OK to have parts of feature panel, which are rendered only sometimes. 

In future we could extend `FeaturedTags` to accomodate this, or even extend iD tagging scheme.